src/test/java/chess/model/position/RankTest.java

```diff
@@ -0,0 +1,24 @@
+package chess.model.position;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import chess.model.piece.Side;
+import chess.model.position.Rank;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class RankTest {
+
+    @ParameterizedTest
+    @CsvSource(value = {"WHITE,TWO,true", "WHITE,FIVE,false", "WHITE,ONE,false", "BLACK,EIGHT,false",
+            "BLACK,SEVEN,true", "BLACK,SIX,false"})
```

Comment:
```suggestion
    @ParameterizedTest
    @CsvSource(value = {
        "WHITE,TWO,true", "WHITE,FIVE,false", "WHITE,ONE,false",
        "BLACK,EIGHT,false", "BLACK,SEVEN,true", "BLACK,SIX,false"})
```
줄바꿈을 잘 활용해서 가독성을 높여보죠!